src/app.module.ts

```diff
@@ -1,13 +1,20 @@
 import { Module } from '@nestjs/common';
+import { ConfigModule } from '@nestjs/config';
 import { MongooseModule } from '@nestjs/mongoose';
 
 import { AppController } from './app.controller';
 import { AppService } from './app.service';
 import { TasksModule } from './tasks/tasks.module';
 
 @Module({
-  imports: [MongooseModule.forRoot('mongodb://localhost/demo'), TasksModule],
+  imports: [
+    MongooseModule.forRoot('mongodb://localhost/demo'),
```

Comment:
if we have already made the env file why not importing the variable by assigning it to process.env and then importing it here instead of manually declaring the same mongo connection string at 2 places.

Reply:
Done now, i did tried to do that but I was not able to achieve it before, but it turns out that the order of importing modules matter in the import statement, but now it's resolved.